### DISCUSSION TITLE
Approved but benefit not available yet.

### DISCUSSION THREAD

**Discussion by yashraj22** [community/community#64715] [GitHub Education] [answered]
### Hi everyone,

![Screenshot_20230823-151314](https://github.com/community/community/assets/29757283/7cc07fed-a050-402c-91c9-46ec843d32d5)
Its saying like this but 3 days passed, do I need to reapply? My friend tell me that he fot instant benefits after approval.

**Comment by arafajs** [comment, marked as answer]
- It may take up to 72 hours for your benefits to be applied after your application is approved.
- If you applied during a busy period, such as the start of a new semester, it may take a bit longer for your benefits to be applied.
- If you uploaded blurry or unclear proof of your academic status, GitHub may need to ask you to provide more clear documentation.
- If it has been more than 72 hours since your application was approved, and you still have not received your benefits, you can try the following:
- Check your spam or junk mail folder. Sometimes, GitHub emails can end up in these folders.
- Go to the GitHub Education Benefits: https://education.github.com/benefits page and click on the "Check Status" button. This will show you the status of your application and benefits.
- If your application is still showing as "Pending" or "In Review", you can contact GitHub support for assistance.

**Reply by 12944qwerty** [reply]
There is no check status button for me.

**Reply by iamjunaidjutt** [reply]
If it is in pending then how to contact Github support for assistance

**Reply by iamjunaidjutt** [reply]
![Screenshot 2024-05-09 002528](https://github.com/community/community/assets/84816593/4a29a78c-2ab6-400e-987c-a3c5759d87b8)
![Screenshot 2024-05-09 002541](https://github.com/community/community/assets/84816593/2dfd07a1-e25d-4963-aba8-f681835bf7eb)

**Comment by smngvlkz** [comment]
Don't worry it will come through. I got access to my benefits today after waiting 3 days. Just be patient

**Comment by Miladlak8** [comment]
Good luck
